docs/02-guides/server-side-events/create-an-event.mdx

```diff
@@ -1,15 +1,158 @@
 ---
 title: Create an event
 description:
-  This guide explains how to create an event in Front-Commerce and dispatch it to external services.
+  In this guide, we will cover the creation of an emitter and how to implement
+  them in your pages, we will create an Emitter that Emit an event when a
+  customer reach a page called FAQ.
```

Comment:
**issue:** This is now wrong after the rework.

```suggestion
  customer votes on an FAQ question.
```

Reply:
Fixed with your suggestion